.github/workflows/acceptance-tests.yml

```diff
@@ -0,0 +1,22 @@
+name: acceptance-tests
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Is it possible to have multiple triggers for an action?  I believe the two actions are identical except for the trigger.

Reply:
Yes, but this affects the status of the overall build. So for example, if a PR fails the "badge" on the readme would turn red. But I agree, there should be a better way to do this as to not duplicate code